tests/rustdoc-ui/unportable-markdown.stderr

```diff
@@ -0,0 +1,61 @@
+error: unportable markdown
+  --> $DIR/unportable-markdown.rs:9:5
+   |
+LL |   /// [^foot]: This is nested within the footnote now, but didn't used to be.
+   |  _____^
+LL | |
+LL | | ///
+LL | | ///     This is a multi-paragraph footnote.
+   | |___________________________________________^
+   |
+   = help: new parser sees Start(Paragraph), old sees End(FootnoteDefinition(Borrowed("foot")))
```

Comment:
I see why it's needed, but it also displays internal stuff. Not sure how helpful it would be for users...